apps/src/music/blockly/blocks/samples.js

```diff
@@ -17,20 +15,16 @@ const isBlockInsideWhenRun = ctx => {
 export const playSound = {
   definition: {
     type: BlockTypes.PLAY_SOUND,
-    message0: '%1 play %2 at measure %3',
+    message0: 'play %1 at measure %2',
     args0: [
       {
-        type: 'field_image',
-        src: getStaticFilePath(PLAY_ICON),
-        width: 18,
-        height: 23,
-        alt: '*',
-        flipRtl: false,
-        name: 'image'
-      },
-      {
-        type: 'input_dummy',
-        name: 'sound'
+        type: 'field_sounds',
+        name: 'sound',
+        getLibrary: () => window.library,
+        playPreview: (id, onStop) => {
+          window.playPreview(id, onStop);
+        },
```

Comment:
I know we're planning on reworking this overall, but I do feel a _little_ hesitant putting these both on `window` and passing them through that way...could we instead maybe have some shared global file that MusicView registers these functions with and share them that way? e.g. some `FieldSoundsHelper`, with `registerMusicPlayer(player)`, `getMusicPlayer()`, `setLibrary`, `getLibrary`, etc. Basically the same idea, but that way it's a little more internal, and it's a little more obvious where this is happening when we revisit this.

Reply:
Good point.  I made a first attempt at a temporary helper for this.